builder-base/Makefile

```diff
@@ -15,8 +15,16 @@ ifeq ($(DEVELOPMENT),true)
 	BASE_IMAGE=amazonlinux:2
 endif
 
+.PHONY: copy-generate-attribution
+copy-generate-attribution:
+	cp -rf ../generate-attribution ./
+
+.PHONY: remove-generate-attribution
+remove-generate-attribution:
+	rm -rf ./generate-attribution
+
 .PHONY: local-images
-local-images:
+local-images: copy-generate-attribution
```

Comment:
Can we just make these as part of the `build` and `release` targets so that we can have something like

```
build: copy-generate-attribution local-images remove-generate-attribution
```

Reply:
good idea!